src/main/dotnet/ProjectFileParser/Jsonify.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using Microsoft.Build.Construction;
+using Microsoft.Build.Evaluation;
+using Newtonsoft.Json.Linq;
+
+namespace ProjectFileParser
```

Comment:
not super confident this works on windows/linux/netfwk combinations, but it'd be super cool to have it work, coz the old code was dirty/hacky as hell !